### PR TITLE
first stab at domain broker migration instructions

### PR DESCRIPTION
**Pull request by bengerman13**

### CODE REVIEW

**Comment by bengerman13** [comment]
fixes 18F/cg-product#1149

File: content/updates/2019-11-01-cdn-domain-service-replacement.md [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+Getting services in org example / space example as me@example.gov...
+
+name                            service                       plan             bound apps             last operation     broker                   upgrade available
+existing-custom-domain          custom-domain                 custom-domain                           create succeeded   domains-broker
+```
+
+The first step is to create a new instance, we'll call it `new-custom-domain` (more detailed instructions available [here]())
+
+```
+$ cf create-service brokername custom-domain new-custom-domain -c '{ "domains": ["www.example.gov"]}'
```
Comment (spgreenberg) [inline content/updates/2019-11-01-cdn-domain-service-replacement.md:27]: brokername should be replaced with service-offering-name. The name of the broker is irrelevant.

File: content/updates/2019-11-01-cdn-domain-service-replacement.md [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+There are no bound apps for this service.
+
+Upgrades are not supported by this broker.
+
+```
+
+The first step is to create the temporary Custom Domain Service instance. We'll call it `temp-custom-domain` in this example:
+
+```
+$ cf create-service brokername custom-domain temp-custom-domain -c '{ "domains": ["www.example.gov"]}'
```
Comment (spgreenberg) [inline content/updates/2019-11-01-cdn-domain-service-replacement.md:180]: brokername should be service-offering-name

File: content/updates/2019-11-01-cdn-domain-service-replacement.md (hunk 1 above)
Comment (bengerman13) [reply content/updates/2019-11-01-cdn-domain-service-replacement.md:27]: yeah - bad verbiage on my part, but the intent is to replace it once we have the name of the new service offering

File: content/updates/2019-11-01-cdn-domain-service-replacement.md [hunk 3]
```diff
@@ -0,0 +1,394 @@
++++
+date = "2019-11-01"
+title = "Replacement of CDN and Custom Domain Service"
++++
+
+We're announcing today the new **name** service broker, providing the new **name** and **name** services, replacing the existing CDN Service and Custom Domain Service, respectively. 
+Due to upstream deprecations, we're forced to deprecate the existing services outside of our normal deprecation policy. Beginning today, the existing CDN Service and Custom Domain service
```
Comment (siennathesane) [inline content/updates/2019-11-01-cdn-domain-service-replacement.md:7]: I think a bit more of detail on the upstream deprecation might help customers, with a link referencing it. Mostly for transparency and to help alleviate any questions along that vein.

**Review by siennathesane — COMMENTED** [review]
For now it's okay but I want to see the new broker code base first.

**Comment by hillaryj** [comment]
FYI this is a PR not a task

**Pull request closed without merging**